### PR TITLE
Feature/ui 10 migrate text input

### PR DESCRIPTION
**Pull request by maciek15**
Merged with master, conflicts in sass configuration from master not accepted from commit https://github.com/PublicInMotionGmbH/ui-kit/commit/7e0997b27e79b617fbf6c4658f3d0f41c6a70c2d

### CODE REVIEW

File: packages/text-input/stories.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+// Load first paragraph from README file
+const readme = getReadmeDescription(require('./README.md'))
+
+// Create factories for story
+const addStory = createStoriesFactory('Text Input', module)
+
+// Stories
+
+const additinalStyling = {
```
Comment (michalochman) [inline packages/text-input/stories.js]: typo

File: packages/text-input/src/TextInput.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  hasError: PropTypes.bool,
+
+  /** Callback for change event */
+  onChange: PropTypes.func,
+
+  /** Input floating label text */
+  placeholder: PropTypes.node,
+
+  /** Size of text input (can be 'small') */
+  size: PropTypes.oneOf(['', 'small']),
```
Comment (michalochman) [inline packages/text-input/src/TextInput.js]: What's the point of empty string? Only to check the `length` property in `render`?

File: packages/text-input/tests/TextInput.test.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      expect(input.props().className).toMatch(/(^| )talixo-text-input--error( |$)/)
+    })
+  })
+
+  describe('when user types inside input', () => {
+    let input
+    const onChange = jest.fn()
+
+    beforeEach(() => {
+      wrapper = shallow(<TextInput placeholder='Test' onChange={onChange} />)
```
Comment (michalochman) [inline packages/text-input/tests/TextInput.test.js]: Why not through `createWrapper`?

File: packages/text-input/src/TextInput.js (hunk 2 above)
Comment (maciek15) [reply packages/text-input/src/TextInput.js]: yes, i'll changed it to `oneOf(['small'])` and check for it like this `size && size.length > 0`

File: packages/text-input/src/TextInput.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+  render () {
+    const {
+      props: { className, hasError, onChange, placeholder, size, style, ...restProps },
+      state: { inputValue },
+      onInputChange
+    } = this
+
+    const inputClasses = cls(moduleName, className, {
+      [`${moduleName}--${size}`]: size && size.length > 0,
```
Comment (michalochman) [inline packages/text-input/src/TextInput.js]: Isn't `size` enough? Empty string evaluates to `false`.

**Pull request merged**